src/app/modules/shared-components/components/progress-section/boolean-section/boolean-section.component.ts

```diff
@@ -1,30 +1,24 @@
-import { Component, ContentChild, EventEmitter, Input, OnInit, Output, TemplateRef } from '@angular/core';
+import { Component, ContentChild, EventEmitter, Input, Output, TemplateRef } from '@angular/core';
 
 @Component({
   selector: 'alg-boolean-section',
   templateUrl: './boolean-section.component.html',
   styleUrls: [ './boolean-section.component.scss' ]
 })
-export class BooleanSectionComponent implements OnInit {
+export class BooleanSectionComponent {
 
   @Input() title = '';
   @Input() icon = '';
 
-  @Input() defaultValue?: boolean;
+  @Input() value = false;
 
   @ContentChild('description') descriptionTemplate?: TemplateRef<any>;
   @ContentChild('label') labelTemplate?: TemplateRef<any>;
 
   @Output() change = new EventEmitter<boolean>();
```

Comment:
Shouldn't you do a two-way binding here? So you would need to call the output `valueChange`. And then you could use more easily this component.
This way, you would not have to do 
```
        [value]="permissions.can_view"
        (change)="this.permissions.can_view = $event"
```  
but just 
```
[(value)]="permissions.can_view"
```

(it may apply to other component as well... to be checked...)

Reply:
fixed, I used two way binding for the progress-section and the boolean-section components